PhysicsTools/NanoAOD/python/electrons_cff.py

```diff
@@ -202,10 +204,20 @@ def _get_bitmapVIDForEle_docstring(modules,WorkingPoints):
     mvaFall17V2noIso = None,
 )
 run2_nanoAOD_94XMiniAODv1.toModify(slimmedElectronsWithUserData.userFloats,
-    ecalTrkEnergyErrPostCorr = cms.InputTag("calibratedPatElectrons94Xv1","ecalTrkEnergyErrPostCorr"),
-    ecalTrkEnergyPreCorr     = cms.InputTag("calibratedPatElectrons94Xv1","ecalTrkEnergyPreCorr"),
-    ecalTrkEnergyPostCorr    = cms.InputTag("calibratedPatElectrons94Xv1","ecalTrkEnergyPostCorr"),
+    ecalTrkEnergyErrPostCorr = cms.InputTag("calibratedPatElectrons94X","ecalTrkEnergyErrPostCorr"),
```

Comment:
I would find it cleaner to just append the New suffix also for v1, as in any case this is just an internal variable, to reduce duplication and customization here and later on

Reply:
either way works, I have opinion here, I'll make the change.